### PR TITLE
Initial code of copy-text-fluxible-plugin

### PR DESCRIPTION
**Pull request by petejodo**
+@robrichard

### CODE REVIEW

**Comment by robrichard** [comment]
I think the convention is to name things fluxible-plugin-*

https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=fluxible%20plugin

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+registry=http://npm.intranet.1stdibs.com:4783
```
Comment (robrichard) [inline .npmrc]: Isn't this is going to be public?

**Comment by PeteJodo** [comment]
whooops. Can we rename repos lol?

File: .npmrc (hunk 1 above)
Comment (petejodo) [inline .npmrc]: oh yeah that was added automatically by `slush-dibs-package`. I'll remove

**Comment by PeteJodo** [comment]
@robrichard I emailed Ross about changing the name and in the mean time I will change the name of the file here in this PR

**Comment by antialias** [comment]
@robrichard where do we stand with respect to javascript 6 in packages? Does it just work? Is `.es.js` required?

**Comment by petejodo** [comment]
@antialias I avoided anything that isn't "default" javascript since I didn't want to finagle with webpack again

**Comment by robrichard** [comment]
I think we should publish compiled versions so the requiring application does not require our specific babel set up

**Comment by antialias** [comment]
> I think we should publish compiled versions so the requiring application does not require our specific babel set up

For making the module available for public use, I'd be on board with that as long the the published versions are built automatically with a script and quarantined in a separate branch that isn't ever included in the master branch.

**Comment by robrichard** [comment]
Convention is to build the files in a prepublish hook and not actually commit them to the repo

https://github.com/1stdibs/react-modal-wrapper/blob/master/package.json#L9

**Comment by petejodo** [comment]
I actually was committing it at the time but I can't remember why I was doing it. Everything builds to the dist folder. I think I did it for people who pull directly from github

**Comment by petejodo** [comment]
The build process for `react-modal-wrapper` is incomplete sort of and I was planning on revisiting it shortly

**Comment by petejodo** [comment]
only because I wanted to commit the built files and it's easy to forget to build before committing but maybe that's not something we want to do anyway because `npm install`ing from a github repo will run the prepublish script anyway I think

**Comment by antialias** [comment]
I guess that'll work, but it's not going to work when the package is set up with `npm link`. And, for a variety of reasons, it would be nice to have the option to develop against the unbuilt source as a consumer of the package.

**Comment by petejodo** [comment]
yeah I agree. I need to flesh that out but I don't think it has any relevance to do with this though

**Comment by antialias** [comment]
> I don't think it has any relevance to do with this though

Agreed. I'm okay with this lot of code since it's not going to break anything.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "nodemon": "^1.8.1",
+    "pre-commit": "^1.1.2"
+  },
+  "dependencies": {
+    "copy-text": "^1.1.0",
+    "fluxible": "^1.0.3",
+    "fluxible-addons-react": "^0.2.2",
+    "lodash.assign": "^3.2.0",
+    "react": "^0.14.5",
+    "react-dom": "^0.14.5"
```
Comment (robrichard) [inline package.json:31]: should react and fluxible be peerDependencies?

Comment (petejodo) [inline package.json:31]: probably but peerDependencies are being removed in npm 3+ so I didn't want to add it but I guess it won't break in later version of npm I think?

Comment (robrichard) [inline package.json:31]: peerDependencies aren't being removed in npm3+, but their behavior is changing. I spoke with @twhid and we think peerDependencies would be appropriate.

http://blog.npmjs.org/post/110924823920/npm-weekly-5

Comment (petejodo) [inline package.json:31]: ok will add them as peer dependencies then

**Comment by petejodo** [comment]
added react and fluxible as peer dependencies

**Comment by petejodo** [comment]
So where are we on this?

**Pull request merged**